app/biotracker/models/target.py

```diff
@@ -0,0 +1,45 @@
+class Target:
```

Comment:
What is the difference between this class and the namedtuple Detection? Like when would we use each?